src/backend/api/Fusion.Resources.Domain/Models/LineOrg/ProfileWithDepartment.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Fusion.Resources.Domain.LineOrg
+{
+    public class ProfileWithDepartment
```

Comment:
Since this is just a DTO for the line org, i'm not really sure the domain project is the correct place for this? Since it is only used one place, i would either use anonymous deserialization or this whole integration could be moved to the application project?